helm/modelix/templates/common/workspace-client-service.yaml

```diff
@@ -11,22 +11,24 @@ spec:
   ports:
   - name: "projector"
     port: 8887
-    targetPort: 8887
-  - name: "modelix-ui"
-    port: 33333
-    targetPort: 33333
+    targetPort: "projector"
   - name: "diff"
     port: 33334
-    targetPort: 33334
+    targetPort: "diff"
   - name: "generator"
     port: 33335
-    targetPort: 33335
-  - name: "debug"
+    targetPort: "generator"
+  {{ if .Values.workspaces.client.experimentalCustomServicePort }}
+  - name: "custom-service"
```

Comment:
Make more then one port exposable, because now more than one port can be exposed.
See https://github.com/modelix/modelix.workspaces/pull/41#discussion_r1551055345